pkg/cmd/cli/cmd/process.go

```diff
@@ -214,14 +212,17 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 		// when user specify the --value
 		if cmd.Flag("value").Changed {
 			values := kcmdutil.GetFlagStringSlice(cmd, "value")
-			injectUserVars(values, cmd, obj)
+			if err := injectUserVars(values, obj); err != nil {
+				return err
```

Comment:
Seems like I'd want a non-zero return code, but that I'd continue on to the next `Template` in my input.